src/components/ERC20Transfers/utils.ts

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+
+export { default } from "./ERC20Transfers";
+
+export const fetchNftData = async () => {
+  try {
+    const response = await axios.get("https://api.polygonscan.com/api", {
+      params: {
+        module: "account",
+        action: "tokentx",
+        address: "0x052D1f06cC98AAe6c4D2e9D06c286143E22dB612",
```

Comment:
@karangorania, the value for this parameter will have to be the state variable that represents the user account.